src/wallet/wallet2.cpp

```diff
@@ -1940,6 +1940,9 @@ bool wallet2::store_keys(const std::string& keys_file_name, const std::string& p
   value2.SetInt(m_merge_destinations ? 1 :0);
   json.AddMember("merge_destinations", value2, json.GetAllocator());
 
+  value2.SetInt(m_testnet ? 1 :0);
```

Comment:
Interesting. I remember looking, and not finding one... It did seem wtf, so maybe I should have looked twice :)